docs/basic-features/eslint.md

```diff
@@ -7,13 +7,13 @@ description: Next.js provides an integrated ESLint experience by default. These
 Since version **11.0.0**, Next.js provides an integrated [ESLint](https://eslint.org/) experience out of the box. To get started, run `next lint`:
 
 ```bash
-next lint
+npx next lint
```

Comment:
Ideally I'd want to recommend `npm run lint` or `yarn run lint` to ensure it's always using the Next.js version that is installed in the project. This would mean we'd create a project with `"lint": "next lint"` in scripts as well

Reply:
Ah good point, [updated!](https://github.com/vercel/next.js/pull/25895/commits/cac5e349aea449b69b8f6745ab35d1b8af718434)